molecule/src/molecule_ionize.cpp

```diff
@@ -444,433 +419,423 @@ void MoleculePkaModel::_loadSimplePkaModel()
     _model.simple_model_ready = true;
 }
 
-void MoleculePkaModel::_loadAdvancedPkaModel()
+namespace  // only for data
```

Comment:
The comment is vague (especially it includes a function). Better specify the purpose or remove any comment.